docs/usage.md

```diff
@@ -95,9 +96,17 @@ while `-vv` will also show `[DEBUG]` messages.
 
 Create a new `treefmt.toml`.
 
+### `-u --on-unmatched`
+
+Log paths that did not match any formatters at the specified log level. Possible values are debug,info,warn,error,fatal.
```

Comment:
```suggestion
### `-u --on-unmatched <debug|info|warn|error|fatal>`

Log paths that did not match any formatters at the specified log level. fatal will exit the formatting with an error. [default: debug]
```

Reply:
I'm trying to keep these descriptions matching the help from the CLI as eventually I want to generate this page instead of manually maintaining it. I'll update the CLI help description to make it a bit clearer.